modules/appeals_api/app/models/appeals_api/appellant.rb

```diff
@@ -129,10 +142,34 @@ def ssn_last_four
       ssn&.last(4)
     end
 
+    def timezone
+      form_data&.dig('timezone').presence&.strip
+    end
+
+    def submission_email_identifier
+      email_id = "#{type.downcase}_email"
+
+      return { id_type: email_id, id_value: email } if email.present?
+
+      icn = mpi_veteran.mpi_icn
+
+      return { id_type: 'ICN', id_value: icn } if icn.present?
+    end
+
+    def signing_appellant?
+      return true if type == 'Claimant' && form_data.present?
+
+      type == 'Veteran' && auth_headers.exclude?('X-VA-Claimant-Last-Name')
+    end
+
     private
 
     attr_accessor :auth_headers, :form_data
 
+    def header_prefix
+      type == 'Veteran' ? '' : '-Claimant'
```

Comment:
You may as well memoize this since it's called in so many other methods we're likely to use.

Also, any reason the `type` needs to be a string? Not a blocker, but I don't see anything that indicates this couldn't be passed into the initializer as a symbol.

Reply:
I never even thought to pass it as a symbol. Going to memoize and look into benefits of passing a symbol.